.github/workflows/conformance-datapath.yaml

```diff
@@ -168,21 +168,24 @@ jobs:
             until docker manifest inspect quay.io/${{ github.repository_owner }}/$image:${{ steps.vars.outputs.sha }} &> /dev/null; do sleep 45s; done
           done
 
-      - name: Derive kube-proxy requirements
+      - name: Derive Cilium feature requirements
         shell: bash
         run: |
             if [[ ${{ matrix.kernel }} == "4.19-"* ]]; then
               # Only kube-proxy
               echo 'KUBE_PROXY=iptables' >> $GITHUB_ENV
               echo 'KPR=disabled' >> $GITHUB_ENV
+              echo 'ENCRYPTION=ipsec' >> $GITHUB_ENV
             elif [[ ${{ matrix.kernel }} == "5.4-"* ]]; then
               # Mixed mode
               echo 'KUBE_PROXY=iptables' >> $GITHUB_ENV
               echo 'KPR=strict' >> $GITHUB_ENV
+              echo 'ENCRYPTION=disabled' >> $GITHUB_ENV
             else
               # Only kube-proxy-replacement
               echo 'KUBE_PROXY=none' >> $GITHUB_ENV
               echo 'KPR=strict' >> $GITHUB_ENV
+              echo 'ENCRYPTION=wireguard' >> $GITHUB_ENV
```

Comment:
See https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs#expanding-or-adding-matrix-configurations to improve this.

Reply:
Unfortunately, I do not see how the GHA matrix config feature can be useful for this case. The main problem is that each permutation is going to be run on a separate runner in parallel which I want to avoid due to cost reasons.

Reply:
You are already using a matrix. I'm not suggesting to add more permutations. I'm suggesting to extend the matrix, while keeping the same number of permutations (5), using `include`:
```yaml
strategy:
  matrix:
    include:
      - kernel: 4.19-main
        encryption: ipsec
      - kernel: 5.4-main
        encryption: disabled
      ...
```
You can do that with `kube_proxy` and `kpr` as well of course.

Reply:
TIL, thanks! Let me try this.